src/themes/oneui/base-variables/_typography.scss

```diff
@@ -1,64 +1,66 @@
 // fonts sizes
 $font-size-base: 14px !default;
-$font-size-small: .8 * $font-size-base !default;
+$font-size-small: 0.8 * $font-size-base !default;
 $font-size-normal: 1 * $font-size-base !default;
 $font-size-large: 1.2 * $font-size-base !default;
 
-$line-height-small:     1.4 * $font-size-small !default;
-$line-height-normal:    1.4 * $font-size-normal !default;
-$line-height-large:     1.4 * $font-size-large !default;
+$line-height-small: 1.4 * $font-size-small !default;
+$line-height-normal: 1.4 * $font-size-normal !default;
+$line-height-large: 1.4 * $font-size-large !default;
 
-$h1-font-size: 1.8 * $font-size-base !default;
-$h2-font-size: 1.6 * $font-size-base !default;
-$h3-font-size: 1.4 * $font-size-base !default;
-$h4-font-size: 1.2 * $font-size-base !default;
-$h5-font-size: 1.0 * $font-size-base !default;
-$h6-font-size: 0.8 * $font-size-base !default;
+$h1-font-size: 4 + $font-size-base !default;
```

Comment:
This sum looks quite random, you might as well just write `18px` (same for heading sizes below)